masterlist.yaml

```diff
@@ -28814,7 +28814,7 @@ plugins:
     msg:
       - <<: *patchRequiem3rdParty
         subs: [ '[Seg''s Requiem Patches](https://www.nexusmods.com/skyrim/mods/95859)' ]
-        condition: 'active("Requiem.esp") and not active("Sacrosanct Requiem Patch and Rebalance.esp")'
+        condition: 'active("Requiem.esp") and not active("Requiem - Sacrosanct Patch.esp")'
```

Comment:
That msg should also be applied to Sacrosanct, not to the patch itself.

````yaml
  - name: 'Sacrosanct - Vampires of Skyrim.esp'
    msg:
      - <<: *patchRequiem3rdParty
        subs: [ '[Seg''s Requiem Patches](https://www.nexusmods.com/skyrim/mods/95859)' ]
        condition: 'active("Requiem.esp") and not active("(Sacrosanct Requiem Patch and Rebalance|Requiem - Sacrosanct Patch).esp")'
```

Reply:
Not your fault, should have been pick up in the pull that added it.

Reply:
Yeah, I fixed it in both places